Changing default build distances

utoxin opened the pull request:
You asked for some suggested changes to the default reach distances. Here you go. :)

utoxin committed:
Changing default build distances

gabevenberg commented:
seems reasonable enough! i like it.

utoxin commented:
Looked through docs on Radar ranges and such, and it looks like 150 (the place range I chose is about the max normal player radar range, while the interact range is a little more than a full on radar.

I /do/ intend to look at creating the config feature you mentioned in the other issue, so for now I think this is a good range.

gabevenberg commented:
according to my testing, 100 is about the reatch of the screen at full zoom out. (can almost reatch the edge to build with data.raw.player.player.build_distance = 100. 
so, maybye build distance at 50, and reatch at 125 or so?

utoxin commented:
Yeah, that's probably more what I'll do for my personal play style. But I'm going to be working on another PR tonight to actually add a config interface into the in-game options, so this can be tuned easily by the player. :)

jaguilar commented:
The reach and build distances should by default cover the entire zoomed out
ordinary window (equiv behavior to 0.14). Of course, I'm happy if we get
the config API implemented and people can tweak as desired without having
to edit lua files.

On Wed, Apr 26, 2017 at 4:46 PM, Matthew Walker <notifications@github.com>
wrote:

> Yeah, that's probably more what I'll do for my personal play style. But
> I'm going to be working on another PR tonight to actually add a config
> interface into the in-game options, so this can be tuned easily by the
> player. :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jaguilar/long-reach/pull/10#issuecomment-297536199>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwzTMCILzUfDmnJVz_awZR6E2Dyxzuxks5rz60xgaJpZM4NI7u2>
> .
>

utoxin commented:
@jaguilar And with my defaults above they DEFINITELY cover that entire area, plus a bit of extra in the radar view, without allowing interaction with extremely distant areas. :)

jaguilar commented:
OK, sounds good to me. Will review and test when I have my son in bed.
Expect the changes to be merged later tonight.

On Wed, Apr 26, 2017 at 4:52 PM, Matthew Walker <notifications@github.com>
wrote:

> @jaguilar <https://github.com/jaguilar> And with my defaults above they
> DEFINITELY cover that entire are, plus a bit of extra in the radar view,
> without allowing interaction with extremely areas. :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jaguilar/long-reach/pull/10#issuecomment-297537525>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwzTKuVIJGf5Pe2r6YT50-8M-haM0pqks5rz65ygaJpZM4NI7u2>
> .
>

gabevenberg commented:
125 cover the entire view area, and a couple tiles outside.

jaguilar commented on long-reach_0.0.7/data-final-fixes.lua:
```diff
@@ -1,2 +1,5 @@
+-- lets you do things like spam turrets into biter nests, so I think 
+-- building should be shorter range than adding / removing items
+data.raw.player.player.build_distance = 150
+data.raw.player.player.reach_distance = 300
```
150 units is already a perfectly safe distance for spamming turrets in biter nests. For simplicity, please remove the rationale and set these to the same value. So long as it covers the entire viewport, I don't really care what the amount is.

utoxin replied:
I'm about 90% done writing the version that actually has a settings GUI.

jaguilar replied:
OK, sounds good to me. May not review tonight, should do tomorrow. :)

jaguilar reviewed (changes requested):
Please make this change, then I'll test and we can merge.

utoxin committed:
Removing extra comments, and unifying values

This is the simple change to make this more reasonable. A larger change that uses the config system is being worked on.

utoxin commented:
Made that change for now. The config screen update has hit some snags that I'm researching, so this is the simple solution.

utoxin commented:
Made obsolete by #11.
